revscoring/scorers/svc.py

```diff
@@ -41,9 +42,11 @@ def train(self, values_scores, balanced_weight=True):
             counts = {}
             for score in scores:
                 counts[score] = counts.get(score, 0) + 1
-
-            weights = [1/(counts[s]/len(scores)) for s in scores]
-            self.svc.fit(scaled_values, scores, weights)
+            
+            self.weights = {s:1-(counts[s]/len(scores)) for s in counts}
```

Comment:
It makes self.weights accessible later.  This may be useful for re-scaling probability estimates.  Regardless, the weights we use when training should be observable.